src/activity/events/components/EventPosters.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from "prop-types";
+import { Gallery, Item } from "react-photoswipe-gallery";
+import "photoswipe/dist/photoswipe.css";
+import "../events.css";
+
+function EventPosters({ posterImages }) {
+	const images = posterImages;
+	return (
+		<Gallery>
+			<div className="poster-list">
+				{images.map(image => {
+					return (
+						<Item
+							key={image.id}
+							original={image.img}
+							thumbnail={image.img}
+							width="1068"
+							height="1068"
+							alt="poster-img"
+						>
+							{({ ref, open }) => (
+								<input
+									className="poster-img"
+									type="image"
+									ref={ref}
+									onClick={open}
+									src={image.img}
+									alt="poster"
+								/>
+							)}
+						</Item>
+					);
+				})}
+			</div>
+		</Gallery>
+	);
+}
+
+export default EventPosters;
+
+EventPosters.propTypes = {
+	posterImages: PropTypes.arrayOf(PropTypes.string).isRequired,
```

Comment:
Console Warning: 
Failed prop type: Invalid prop `posterImages[0]` of type `object` supplied to `EventPosters`, expected `string`.

Reply:
Console Warning still present: 
Failed prop type: Invalid prop `posterImages[0]` of type `Object` supplied to `EventPosters`, expected instance of `Map`.

(Also, on line 1, { string } isn't being used )

Reply:
I could not see the warning after checking workflows and running lint. Anyways, the code was updated. Let me know if there are issues.

Reply:
It won't show on your terminal, but if you do "npm start" and look at the console of the browser, it'll show